bc_obps/reporting/schema/report_final_review.py

```diff
@@ -388,22 +386,29 @@ def resolve_operation_emission_summary(obj: ReportVersion) -> Optional[dict]:
     def resolve_is_supplementary_report(obj: ReportVersion) -> bool:
         return not ReportVersionService.is_initial_report_version(obj.id)
 
+    class Meta:
+        model = ReportVersion
+        fields = [
+            "report_type",
+            "is_latest_submitted",
+            "reason_for_change",
+            "status",
+        ]
+
+
+class FinalReviewVersionSchema(BaseReportVersionSchema):
+    facility_reports: Union[Dict[str, FacilityReportSchema], List[FacilityReportLFOSchema]] = {}
```

Comment:
I'm not a big fan of having surprise (non-deterministic) types for our variables. Maybe something we can refactor at a later point